exampleoc/structs-0.go

```diff
@@ -1354,6 +1358,94 @@ func (*Model_MultiKey) ΛBelongingModule() string {
 	return "openconfig-withlistval"
 }
 
+// Model_NoKey represents the /openconfig-withlistval/model/c/no-key YANG schema element.
+type Model_NoKey struct {
+	Key   *string `path:"state/key" module:"openconfig-withlistval/openconfig-withlistval"`
+	Value *int64  `path:"state/value" module:"openconfig-withlistval/openconfig-withlistval"`
+}
+
+// IsYANGGoStruct ensures that Model_NoKey implements the yang.GoStruct
+// interface. This allows functions that need to handle this struct to
+// identify it as being generated by ygen.
+func (*Model_NoKey) IsYANGGoStruct() {}
+
+// GetKey retrieves the value of the leaf Key from the Model_NoKey
+// struct. If the field is unset but has a default value in the YANG schema,
+// then the default value will be returned.
+// Caution should be exercised whilst using this method since when without a
+// default value, it will return the Go zero value if the field is explicitly
+// unset. If the caller explicitly does not care if Key is set, it can
+// safely use t.GetKey() to retrieve the value. In the case that the
+// caller has different actions based on whether the leaf is set or unset, it
+// should use 'if t.Key == nil' before retrieving the leaf's value.
+func (t *Model_NoKey) GetKey() string {
+	if t == nil || t.Key == nil {
+		return ""
+	}
+	return *t.Key
+}
+
+// GetValue retrieves the value of the leaf Value from the Model_NoKey
+// struct. If the field is unset but has a default value in the YANG schema,
+// then the default value will be returned.
+// Caution should be exercised whilst using this method since when without a
+// default value, it will return the Go zero value if the field is explicitly
+// unset. If the caller explicitly does not care if Value is set, it can
+// safely use t.GetValue() to retrieve the value. In the case that the
+// caller has different actions based on whether the leaf is set or unset, it
+// should use 'if t.Value == nil' before retrieving the leaf's value.
+func (t *Model_NoKey) GetValue() int64 {
+	if t == nil || t.Value == nil {
+		return 0
+	}
+	return *t.Value
+}
+
+// SetKey sets the value of the leaf Key in the Model_NoKey
+// struct.
+func (t *Model_NoKey) SetKey(v string) {
+	t.Key = &v
+}
+
+// SetValue sets the value of the leaf Value in the Model_NoKey
+// struct.
+func (t *Model_NoKey) SetValue(v int64) {
+	t.Value = &v
+}
+
+// PopulateDefaults recursively populates unset leaf fields in the Model_NoKey
+// with default values as specified in the YANG schema, instantiating any nil
+// container fields.
+func (t *Model_NoKey) PopulateDefaults() {
+	if t == nil {
+		return
+	}
+	ygot.BuildEmptyTree(t)
+}
+
+// Validate validates s against the YANG schema corresponding to its type.
```

Comment:
This doc and the one below refers to the receiver as "s" instead of "t". Does that originate in ygot somewhere? Not a big deal if it's fixed separately or later.

Reply:
yeah, from here https://github.com/openconfig/ygot/blob/master/gogen/gogen.go#L521